.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "make-utils"]
+	path = make-utils
+	url = git@github.counsyl.com:dev/make-utils.git
```

Comment:
Me too. It's not necessary to build this package out in the wild, but it does let us use it in our GoCD template. WDYT? I could copy `versioning.mk` into this, but I like that even less since it can go out of sync and it's not necessary outside of Counsyl.